src/main/java/de/embl/cba/mobie/bdv/ViewerTransformLogger.java

```diff
@@ -0,0 +1,54 @@
+package de.embl.cba.mobie.bdv;
+
+import bdv.util.BdvHandle;
+import com.google.gson.Gson;
+import de.embl.cba.bdv.utils.BdvUtils;
+import de.embl.cba.bdv.utils.Logger;
+import de.embl.cba.mobie.Utils;
+import de.embl.cba.mobie.serialize.JsonHelper;
+import de.embl.cba.mobie.transform.AffineViewerTransform;
+import de.embl.cba.mobie.transform.NormalizedAffineViewerTransform;
+import de.embl.cba.mobie.transform.PositionViewerTransform;
+import net.imglib2.realtransform.AffineTransform3D;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import sc.fiji.bdvpg.scijava.command.BdvPlaygroundActionCommand;
+
+@Plugin( type = BdvPlaygroundActionCommand.class, name = ViewerTransformLogger.NAME, menuPath = ViewerTransformLogger.NAME )
+public class ViewerTransformLogger implements BdvPlaygroundActionCommand
+{
+	public static final String NAME = "BDV - Log Current View";
+
+	@Parameter
+	BdvHandle bdvh;
+
+	@Override
+	public void run()
+	{
+		new Thread( () -> {
+
+			// position
+			final PositionViewerTransform positionViewerTransform = new PositionViewerTransform( BdvUtils.getGlobalMouseCoordinates( bdvh ).positionAsDoubleArray() );
+
+			// affine
+			final AffineTransform3D affineTransform3D = new AffineTransform3D();
+			bdvh.getViewerPanel().state().getViewerTransform( affineTransform3D );
+			final AffineViewerTransform affineViewerTransform = new AffineViewerTransform( affineTransform3D.getRowPackedCopy() );
+
+			// normalized affine
+			final AffineTransform3D normalisedViewerTransform = Utils.createNormalisedViewerTransform( bdvh, Utils.getMousePosition( bdvh ) );
```

Comment:
@tischi I wonder if we should use the central point of the viewer for this? Rather than ```Utils.getMousePosition()```. The issue is that we are doing this via a right click menu, so your mouse position depends on how far down in the list the command to log the view is.

Reply:
Good idea! I can change it...